### ISSUE TITLE
beta channel not being updated

### ISSUE THREAD

**Issue by teohhanhui** [flathub/com.google.Chrome#185]
It's still on version 101. That's really old by now.

Is the `beta` channel abandoned?

**Comment by refi64** [comment]
The beta version was only for testing before it was on Flathub stable, I guess I should see if it can be EOL'd while leaving the stable branch intact?

**Comment by teohhanhui** [comment]
Right... It'd be nice to have beta channel of Google Chrome though.

**Comment by wjt** [comment]
https://flathub.org/apps/details/com.google.ChromeDev ?
